content/en/security/severity_scoring.md

```diff
@@ -0,0 +1,116 @@
+|---------------|---------------------|---------------|----------|
+|               |                     | **Private**       | **Public**   |
+| **Attack Vector** | **Required Privileges** | Low           | Medium   |
+|               | **Vulnerability**       | Medium        | High     |
+|               | **No Authorization**    | High          | Critical |
+
+The attack vector is determined by the following criteria:
+
+|    Attack Vector    |                                                 Definition                                                |
+|:-------------------:|:---------------------------------------------------------------------------------------------------------:|
+| Required Privileges | Requires specific privileges or access to abuse.                                                          |
+| Vulnerability       | Requires a vulnerable component to abuse, such as a software vulnerability on a compute instance or a leaked password/access key. |
+| No Authorization    | Requires no authorization/authentication to abuse.                                                        |
+
+The accessibility is determined by the following criteria:
+
+| Accessibility |                             Definition                             |
+|:-------------:|:------------------------------------------------------------------:|
+| Private       | The vulnerable component/resource is in a private network.         |
+| Public        | The vulnerable component/resource is accessible from the internet. |
+
+#### Impact
+
+The impact component is how damaging the exploitation of the misconfiguration would be to the environment.
+
+|  Impact  |                                                                                                                 Definition                                                                                                                |
+|:--------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|    Low   | This misconfiguration is related to security hardening, hygiene, resource metadata, or industry best practice configurations. By itself, this misconfiguration represents little to no impact to the environment.                                                                                                             |
+|  Medium  | Abusing this misconfiguration results in an impact to the confidentiality, integrity, or availability of the vulnerable component or its directly associated resources.                                                                   |
+|   High   | Abusing this misconfiguration results in an impact to the following: confidentiality, integrity or availability of the vulnerable component and impacts a significant number of other resources (E.G. S3FullAccess, EC2FullAccess, etc.). |
```

Comment:
the sentence makes it sound like S3FullAccess is "another resource", would rephrase slightly and use inline code blocks for the policy names

Reply:
Changed the language and added the `code block` where needed.